app/src/main/kotlin/io/homeassistant/companion/android/themes/NightModeManager.kt

```diff
@@ -0,0 +1,37 @@
+package io.homeassistant.companion.android.themes
+
+import android.os.Build
+import io.homeassistant.companion.android.common.data.prefs.NightModeTheme
+import io.homeassistant.companion.android.common.data.prefs.PrefsRepository
+import javax.inject.Inject
+import timber.log.Timber
+
+class NightModeManager @Inject constructor(private val prefsRepository: PrefsRepository) {
```

Comment:
I might be wrong but for me it is mostly focus on whether or not it's in night mode?